hack/local-up-cluster.sh

```diff
@@ -665,8 +688,19 @@ fi
 
 if [[ "${START_MODE}" != "kubeletonly" ]]; then
   echo
-  cat <<EOF
-To start using your cluster, open up another terminal/tab and run:
+  if [[ -n "${LOCAL_KUBECONFIG_PATH}" ]]; then
+    ${KUBECTL} --kubeconfig=${LOCAL_KUBECONFIG_PATH} config set-cluster local --server=https://${API_HOST}:${API_SECURE_PORT} --certificate-authority=${ROOT_CA_FILE}
+    ${KUBECTL} --kubeconfig=${LOCAL_KUBECONFIG_PATH} config set-credentials myself ${AUTH_ARGS}
+    ${KUBECTL} --kubeconfig=${LOCAL_KUBECONFIG_PATH} config set-context local --cluster=local --user=myself
+    ${KUBECTL} --kubeconfig=${LOCAL_KUBECONFIG_PATH} config use-context local
+  else
+    cat <<EOF
+To start using your cluster, you can open up another terminal/tab and run:
+
+  export KUBECONFIG=/var/run/kubernetes/admin.kubeconfig
```

Comment:
use ${CERT_DIR}, don't hard-code /var/run/...

@deads2k you have opinions about how we should tell people to use this kubeconfig?